guideline/cfh.md

```diff
@@ -205,12 +190,6 @@ The proposal must cover all of the following points:
    5. Conference desk manager: responsible for the on-site help desk, on-site
       registrations, tickets, cash refunds, attendee support, etc.
 
-6. List of all team members that will be present during the conference itself
```

Comment:
It's good to know who in the team will physically be present.  It will help PAC to find contact windows.